torchmdnet/datasets/spice.py

```diff
@@ -97,7 +97,7 @@ def sample_iter(self):
                 * self.HARTREE_TO_EV
             )
             all_dy = (
-                pt.tensor(mol["dft_total_gradient"], dtype=pt.float32)
+                -pt.tensor(mol["dft_total_gradient"], dtype=pt.float32)
```

Comment:
From the HDF5 loader i understand that there is a group["forces"] in the SPICE dataset, so why you use dft_total_gradient?

Reply:
They're totally different formats.  They just happen to both use HDF5.